api/src/main/java/me/lokka30/treasury/api/common/event/EventCaller.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * This file is/was part of Treasury. To read more information about Treasury such as its licensing, see <https://github.com/lokka30/Treasury>.
+ */
+
+package me.lokka30.treasury.api.common.event;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.Set;
+import java.util.TreeSet;
+import org.jetbrains.annotations.NotNull;
+
+class EventCaller {
```

Comment:
missing javadoc

Reply:
thats an implementation detail ; we don't need javadoc